hasura/migrations/default/1705344968980_modify-app-metadata-functions/up.sql

```diff
@@ -0,0 +1,39 @@
+ALTER TABLE "public"."app_metadata"
+ADD COLUMN unique_verification_status_row varchar NOT NULL UNIQUE;
+
+CREATE OR REPLACE FUNCTION validate_single_url(url text)
+RETURNS void as $$
+BEGIN
+  IF url IS NOT NULL AND url != '' THEN
+--   Allow hash params now
+    IF NOT (url ~* '^https://([[:alnum:]_-]+\.)+[[:alnum:]_-]+(/[[:alnum:]_\-./?%&#=]*)?$') THEN
+      RAISE EXCEPTION USING ERRCODE= '22000', MESSAGE= 'Invalid URL format. URLs must use HTTPS protocol.';
+    END IF;
+  END IF;
+END;
+$$ LANGUAGE plpgsql;
+
+DROP TRIGGER IF EXISTS "trigger_enforce_app_id_row_limit" ON "public"."app_metadata";
+
+DROP FUNCTION IF EXISTS "enforce_app_id_row_limit";
+
+CREATE FUNCTION set_unique_verification_status_row()
+RETURNS TRIGGER AS $$
+BEGIN
+  IF NEW.status = 'verified' THEN
+    NEW.unique_verification_status_row := NEW.app_id || '_verified';
```

Comment:
suggest changing to `can_row_be_edited` for clarity or something along those lines

Reply:
changed to is_row_verified to more accurately reflect it's value